cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -485,7 +485,7 @@ func (m *gceManagerImpl) GetMigTemplateNode(mig Mig) (*apiv1.Node, error) {
 	if err != nil {
 		return nil, err
 	}
-	return m.templates.BuildNodeFromTemplate(mig, template, cpu, mem)
+	return m.templates.BuildNodeFromTemplate(mig, template, cpu, mem, nil)
```

Comment:
Is pods always going to be nil then? It doesn't seem it's possible to actually set this by configuring the mig...